.github/workflows/docker-build.yaml

```diff
@@ -73,16 +84,17 @@ jobs:
     name: process-image-manifest
     needs: ["setup", "build"]
     steps:
-      - name: Log in to Registry
-        uses: redhat-actions/podman-login@632d91dfe19e1b55833cb708786bfbad2c2a0335 #pin@v3.2.0
+      - name: Log in to Quay.io
```

Comment:
We are not using quay yet. We want to support multiple registries and so far we support only GHCR. See #13